core/src/main/java/io/kestra/core/schedulers/AbstractScheduler.java

```diff
@@ -101,8 +98,6 @@ public void run() {
             TimeUnit.SECONDS
         );
 
-        flowListeners.listen(this::computeSchedulable);
```

Comment:
this must be keep !! 
If not, you will receive only update, not all the previous flow